docs/installation/server-config/index.rst

```diff
@@ -195,53 +200,50 @@ The default number of partitions for the topics created by KSQL. The default is
 ksql.sink.replicas
 ^^^^^^^^^^^^^^^^^^
 
-The default number of replicas for the topics created by KSQL. The default is one. 
+The default number of replicas for the topics created by KSQL. The default is one.
 
 .. _ksql-listeners:
 
 ^^^^^^^^^
 listeners
 ^^^^^^^^^
 
-Set the port for the KSQL Server to listen on. This defaults to http://localhost:8088. To listen for on port 80, update it accordingly
-in your ``ksql-server.properties`` file.
+Set the port for the KSQL Server REST API to listen on. This defaults to http://localhost:8088. To listen on port ``80``, update it accordingly in your ``ksql-server.properties`` file.
 
 .. code:: bash
 
     # Set which port to listen on.
-    listeners=http://0.0.0.0:80 
-    
-.. _restrict-ksql-interactive:
+    listeners=http://0.0.0.0:80
 
-----------------------------------
-Restricting Interactive KSQL Usage
-----------------------------------
+.. _restrict-ksql-interactive:
 
-KSQL supports locked-down deployment scenarios where you can restrict interactive use of the KSQL cluster.
+------------------------------------
+Headless, non-interactive KSQL Usage
```

Comment:
shouldn't use commas in the title, and should be capitalized to match other titles (e.g. `Headless Non-interactive KSQL Usage`). on a side note - is the term `headless` known in this context?